.internal-ci/helm/fog-ingest/values.yaml

```diff
@@ -30,25 +30,26 @@ fogIngest:
   podAnnotations:
     fluentbit.io/include: 'true' # collect logs with fluentbit
     fluentbit.io/exclude-jaeger-agent: 'true'
+    sgx.intel.com/quote-provider: fog-ingest
```

Comment:
What does this do? 

`quote-provider` makes me think its the source for getting quotes, so Azure, or Intel endpoint.